lib/spack/spack/cmd/verify.py

```diff
@@ -74,7 +73,7 @@ def verify(parser, args):
 
     elif args.specs_or_files:
         # construct disambiguated spec list
-        env = ev.get_env(args, 'verify')
+        env = spack.environment.get_active_env()
```

Comment:
@haampie I think this might be related to the test failure -- you can get some really inscrutable errors with improper imports.

Reply:
Hm okay, let me try to fix the circular dependencies. Note that `def matching_spec_from_env` in `__init__.py` already used `spack.environment.<stuff>` without importing before this pr.

Reply:
This commit 40e790f9049a5da2956327aa86b17a14eb3f19ec avoids circular import errors, is that OK for you @alalazo? Seems like other methods already did a local import of spack.compiler(s) before (see `def optimization_flags`).

Only after that it's OK to add those `spack.environment` imports in spack.cmd.*, I've done it over the whole code base in 0d208342b433b6da11c48aee9f6d4209aee05dc3, and made it consistently use `import spack.environment as ev` since this was already done in 55 out of 62 cases.